### ISSUE TITLE
Which license?

### ISSUE THREAD

**Issue by detlefm** [artnoage/Podcast#2]
What license is the code in this repository under?

**Comment by artnoage** [comment]
You are right, I forgot to add one. I put an Apache one. feel free to do whatever you like with it. If you have any questions I will be glad to help
